documents/converters.go

```diff
@@ -433,3 +434,105 @@ func convertNFTs(tokenRegistry TokenRegistry, nfts []*coredocumentpb.NFT) (nnfts
 	}
 	return nnfts, err
 }
+
+// toP2PAttributes convert model attributes to p2p attributes
+// since the p2p respresentation of attributes is a list, we will always sort the keys and then insert to the list.
+func toP2PAttributes(attrs map[AttrKey]Attribute) (pattrs []*coredocumentpb.Attribute, err error) {
```

Comment:
```suggestion
func toProtocolAttributes(attrs map[AttrKey]Attribute) (pattrs []*coredocumentpb.Attribute, err error) {
```
I would like to use the word `protocol` here, as it is a protocol format that is used in P2P communications.